paint/example/src/lib.rs

```diff
@@ -482,6 +482,15 @@ decl_module! {
 			Ok(())
 		}
 
+		// TODO: this stuff will move to a new test.
+		#[weight = sr_primitives::weights::FunctionOf(
+			|args: (&u32, &T::AccountId, &BalanceOf<T>)| (args.0 + 100) as Weight,
+			1u8,
```

Comment:
Can we do this?
```suggestion
			DispatchClass::Operational,
```

Reply:
You can use any type here that implements `Into<DispatchClass>`. We can make a balnket impl for this. My general impression was that we prefer number encoding for brevity.